src/litdata/processing/utilities.py

```diff
@@ -219,30 +215,30 @@ def read_index_file_content(output_dir: Dir) -> Optional[Dict[str, Any]]:
             return json.load(f)
 
     else:
-        # download the index file from s3, and read it
+        # download the index file from the cloud provider, and read it
         obj = parse.urlparse(output_dir.url)
 
-        if obj.scheme != "s3":
-            raise ValueError(f"The provided folder should start with s3://. Found {output_dir.path}.")
+        if obj.scheme not in _SUPPORTED_PROVIDERS:
+            raise ValueError(f"The provided folder should start with {_SUPPORTED_PROVIDERS}. Found {output_dir.path}.")
```

Comment:
```suggestion
            raise ValueError(f"The provided folder should start with one of {[el + '://' for el in _SUPPORTED_PROVIDERS]}. Found {output_dir.path}.")
```

might be good to have a small utility function

Reply:
sure. Thanks.